SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day3/arraylist/ExerciseArraylist.java

```diff
@@ -0,0 +1,44 @@
+package hackerrank.day3.arraylist;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 31/08/21
+ */
+public class ExerciseArraylist {
+
+    public static final String ERROR_MESSAGE = "ERROR!";
+
+    public static void getArrayInfo() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        int numberLines = scanner.nextInt();
+
+        List<List> dynamicTable = new  ArrayList<>();
+
+        for (int tableIndex = 0; tableIndex < numberLines; tableIndex++) {
+            int arraySize = scanner.nextInt();
+            ArrayList<Integer> arrayElements = new ArrayList<>();
```

Comment:
Also remember Code for Interface, not for Implementation:
https://www.javaguides.net/2018/06/java-collection-framework-best-practices.html